python/ray/exceptions.py

```diff
@@ -311,6 +313,9 @@ def __init__(self, cause: Union[RayTaskError, ActorDiedErrorContext] = None):
                 error_msg_lines.append(
                     "The actor never ran - it was cancelled before it started running."
                 )
+            if cause.preempted:
+                self.preempted = True
+                error_msg_lines.append("\tThe actor's node was preempted.")
```

Comment:
Are we going to show the actual preempt reason? Spot or fallback?

Reply:
not in the scope of this PR. Is this reported as drain reason?

Reply:
Yes.

Reply:
This information is not currently reported from autoscaler.